lib/dispatch-rider/logging/base_formatter.rb

```diff
@@ -0,0 +1,30 @@
+module DispatchRider
+  module Logging
+    class BaseFormatter
+      def format_error_handler_fail(message, exception)
+        raise NotImplementedError
+      end
+
+      def format_got_stop(message, reason)
```

Comment:
Unused method argument - `message`. If it's necessary, use `_` or `_message` as an argument name to indicate that it won't be used. You can also write as `format_got_stop(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `reason`. If it's necessary, use `_` or `_reason` as an argument name to indicate that it won't be used. You can also write as `format_got_stop(*)` if you want the method to accept any arguments but don't care about them.